fbw-common/src/systems/instruments/src/OANC/BrakeToVacateUtils.ts

```diff
@@ -328,7 +328,8 @@ export class BrakeToVacateUtils<T extends number> {
     this.btvExitPosition = btvExitPosition;
 
     // Account for touchdown zone distance
-    const correctedStoppingDistance = reqStoppingDistance - MIN_TOUCHDOWN_ZONE_DISTANCE;
+    // Don't subtract touchdown zone distance for now, creates confusion for users
+    const correctedStoppingDistance = reqStoppingDistance;
```

Comment:
This should be either correct or not. Doesn't really matter if people are confused when it works like the real thing.

Reply:
This is for the Navigraph-less BTV fallback, where users enter a braking distance manually. So there's no real thing anyways, this line is just about how we want to define the LDA or rather where the BTV stop target is.

Before: braking distance was measured from 400m, i.e. static TDZ distance
After: Braking distance now measured from threshold, far easier to read from charts from my POV

I'd say after the change we're more correct, since it resembles the definition of LDA (beginning at the threshold)

Reply:
improved the comment now

Reply:
Btv, I removed this change from here, I refactored most of the code here: https://github.com/flybywiresim/aircraft/pull/9355